enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -77,8 +97,6 @@ java -jar metabase.jar
 
 ## Authenticate users from your back-end
 
-> **Note:** Metabase Embedding SDK for React only supports JWT authentication.
```

Comment:
I'd rather you keep this sentence but expand it to like 

> **Note:** Metabase Embedding SDK for React only supports JWT authentication in production. And we also support API Key authentication in development.

Or something, you can polish the sentence to your liking